Pipfile.lock

```diff
@@ -9,12 +9,12 @@
             "os_name": "posix",
             "platform_machine": "x86_64",
             "platform_python_implementation": "CPython",
-            "platform_release": "4.15.14-300.fc27.x86_64",
-            "platform_system": "Linux",
-            "platform_version": "#1 SMP Thu Mar 29 16:13:44 UTC 2018",
+            "platform_release": "17.5.0",
+            "platform_system": "Darwin",
+            "platform_version": "Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 PST 2018; root:xnu-4570.51.1~1/RELEASE_X86_64",
             "python_full_version": "3.6.5",
             "python_version": "3.6",
-            "sys_platform": "linux"
+            "sys_platform": "darwin"
```

Comment:
@goern is this a problem? Looks like you've frozen you pip env on a mac but we'll deploy on linux

Reply:
thats not a problem to the S2I on OpenShift GNU/Linux